CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## [0.33.0]
 
+- The first parameter of all `monaco.languages.register*Provider` functions have changed to take a `DocumentSelector` instead of a single `languageId`
+- The `Environment.getWorker` function can now return a `Promise`
```

Comment:
has changed ;)

Reply:
I have no idea what you are talking about :disappointed_relieved: :smile: 

(Shame on me)

Reply:
(amended)